src/ui/components/UploadScreen/Privacy.tsx

```diff
@@ -83,7 +83,7 @@ function PrivacyData({ icon, name, urls }: { icon: string; name: string; urls: s
 
 export function Privacy() {
   const { recording } = useGetRecording(getRecordingId());
-  const { operations } = recording!;
+  const { operations } = recording ?? {};
```

Comment:
I think there's no guarantee that `recording` won't be undefined (still `loading`), in which case this component would fall down.